rulesets/standard.json

```diff
@@ -63,6 +63,11 @@
             ]
         }
     },
+    "positive_decimal": {
+        "cases": [ {
+            "paths": [ "recipient-country/@percentage", "recipient-region/@percentage", "sector/@percentage", "capital-spend/@percentage" ]
+         } ]
+    },
```

Comment:
This bit doesn’t look right to me – I think the curly bracket on L65 should be moved down a bit.

The `positive_decimal` rule is currently at the same level as `//iati-activity` and `//iati-organisation` (the `context_xpath`). `Rule`s should be one level down from these `context_xpath`s.

Reply:
I think it’s correct in meta_tests/positive_decimal.json, but wrong here.